bukkit/src/main/java/com/github/games647/fastlogin/bukkit/task/DelayedAuthHook.java

```diff
@@ -94,13 +94,13 @@ private boolean registerHooks() {
 
     private AuthPlugin<Player> getAuthHook() {
         try {
-            @SuppressWarnings("unchecked")
             List<Class<? extends AuthPlugin<Player>>> hooks = Arrays.asList(AuthMeHook.class,
                     CrazyLoginHook.class, LogItHook.class, LoginSecurityHook.class,
                     SodionAuthHook.class, UltraAuthHook.class, xAuthHook.class);
 
             for (Class<? extends AuthPlugin<Player>> clazz : hooks) {
-                String pluginName = clazz.getSimpleName().replace("Hook", "");
```

Comment:
I prefer to keep it explicit or make it `...length() - "Hook".length`. Otherwise it's a magic number, where the origin is not really obvious.